pkg/apis/serving/fieldmask.go

```diff
@@ -183,6 +183,7 @@ func PodSpecMask(ctx context.Context, in *corev1.PodSpec) *corev1.PodSpec {
 	out.Volumes = in.Volumes
 	out.ImagePullSecrets = in.ImagePullSecrets
 	out.EnableServiceLinks = in.EnableServiceLinks
+	out.AutomountServiceAccountToken = in.AutomountServiceAccountToken
```

Comment:
I thought I remembered we were going to have this only allow `false`, since `true` could theoretically make things less secure (if an administrator had set `AutomountServiceAccountToken` = false on the service account), but looks like that was never written on the issue 🤔. TBH maybe this is fine, though, interested in opinions @dprotaso @markusthoemmes

Reply:
Let's just allow setting this to false

Reply:
ack